Formalisation(shacl)/Core/PiecesShape/DataService.ttl

```diff
@@ -0,0 +1,37 @@
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix dcat: <http://www.w3.org/ns/dcat#> .
+@prefix dct: <http://purl.org/dc/terms/> .
+@prefix dash: <http://datashapes.org/dash#> .
+
+
+:DataServiceShape a sh:NodeShape ;
+  sh:targetClass dcat:DataService ;
+  sh:property [
+    sh:path dcat:endpointURL ;
+    sh:nodeKind sh:IRI ;
+    sh:minCount 1 ;
+    sh:name "Endpoint URL" ;
+    dash:editor dash:URIEditor ;
+    dash:viewer dash:URIViewer ;
+  ],
+  [
+    sh:path dct:title ;
+    sh:nodeKind sh:Literal ;
+    dash:editor dash:TextFieldEditor ;
+    dash:viewer dash:LabelViewer ;
+    sh:minCount 1 ;
+    sh:uniqueLang true ;
+  ],
+  [
+    sh:path dct:endpointDescription ;
+    sh:nodeKind sh:Literal ;
+    dash:editor dash:TextFieldEditor ;
+    dash:viewer dash:LabelViewer ;
```

Comment:
```suggestion
    dash:viewer dash:LiteralViewer ;
```

Reply:
the commitlint test failed on the commit message.